src/components/CategoryCard/CategoryCard.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
```

Comment:
This really shouldn't have been included in this commit - its more linked to your next task.